src/transactions.ts

```diff
@@ -81,6 +86,30 @@ export class ReadTransactionImpl implements ReadTransaction {
     return result['has'];
   }
 
+  async size(): Promise<number> {
```

Comment:
Split on:

`size(): Promise<number>`

vs

`get size(): Promise<number>`

Reply:
This is so expensive way to get this. Let's just leave it out until we can do it right?

Reply:
Oh and personally I prefer size() if it's going to do an RPC.